package.json

```diff
@@ -44,8 +44,10 @@
     "react": "15.1.0",
     "react-bootstrap": "0.29.4",
     "react-dom": "15.1.0",
+    "react-redux": "4.4.5",
     "react-router": "2.4.1",
-    "react-tap-event-plugin": "^1.0.0"
+    "react-tap-event-plugin": "^1.0.0",
```

Comment:
Jeg foreslår at vi fjerne alle hatter ^ foran versjonsnummer, slik at applikasjonen vår ikke plutselig brekker når noen oppdatererer tredjepartsbiblioteket sitt.

Forøvrig anbefaler jeg at du kjører
```
npm config set save-exact true -g
```
så slipper du gå inn og fjerne hatten manuelt i fremtiden.